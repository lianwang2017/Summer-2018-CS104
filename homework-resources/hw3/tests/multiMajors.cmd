CSCI
HRTHS
BUAD
EE
FNDR
CANST
PHCS