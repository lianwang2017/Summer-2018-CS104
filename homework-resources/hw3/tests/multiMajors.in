Aaron Cote, CSCI HRTHS
Max Nikias, BUAD EE BUAD FNDR
Steven B. Sample, BUAD EE
David Pritchard, CANST CSCI PHCS