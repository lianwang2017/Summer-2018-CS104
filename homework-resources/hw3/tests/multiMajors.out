CSCI 
Aaron Cote
David Pritchard

HRTHS 
Aaron Cote

BUAD 
Max Nikias
Steven B. Sample

EE 
Max Nikias
Steven B. Sample

FNDR 
Max Nikias

CANST 
David Pritchard

PHCS 
David Pritchard

